docs/getstarted.md

```diff
@@ -1,9 +1,9 @@
 # Get started
 ## Data processing
 
-### Graphs generation
+### Data mapping
 
-The process of generating graphs takes as input `.pdb` files representing protein-protein structural complexes and the correspondent Position-Specific Scoring Matrices (PSSMs) in the form of `.pssm` files. Query objects describe how the graphs should be built.
+For each protein-protein complex, a query can be created and added to the `QueryCollection` object, to be processed later on. Different types of queries exist, based on the molecular resolution needed (`ProteinProteinInterfaceResidueQuery`, `ProteinProteinInterfaceAtomicQuery`). Each query takes as input a `.pdb` file, representing the protein-protein structural complex, the ids of the two chains composing the complex, and the correspondent Position-Specific Scoring Matrices (PSSMs), in the form of `.pssm` files.
```

Comment:
see my suggestion in the readme (isn't there a way to just reference the readme section rather than duplicating it here?)

Reply:
I just copy paste the right sections when I modify the readme. The get started contains more sections than the readme (this is the biggest difference), and I bet there is a way to just import those extra sections (maybe from a md file) and using the readme in docs/ al well file but I should look for it. For now, I think it's fine as it is, we only need to remember to modify them "in pairs"